### DISCUSSION TITLE
TypeError: issubclass() arg 2 must be a class or tuple of classes

### DISCUSSION THREAD

**Discussion by luboskriss** [optuna/optuna#5712] [Q&A]
I am trying to use Optuna to optimize various normalizations and explainable methods from Captum based on various understandability and fidelity techniques. During the computation of explanations, a warning from captum library pops up and does not continue. I got this error not related to the warning problem. Unfortunately, the code is not reproducible and shareable. Does anyone have this type of experience?

**Comment by nzw0301** [comment]
I'm afraid that we can't answer your question since the script was not shared at all and only vague error message.

**Comment by nzw0301** [comment]
Please feel free to reopen the discussion if you can share more details.
